meerk40t/device/ruida/ruidadevice.py

```diff
@@ -173,6 +173,7 @@ def __init__(self, context, path):
         self.filestream = None
 
         self.cutcode = CutCode()
+        self.plotcut = PlotCut()
```

Comment:
All cutcode entries were merged into plotcode calls. Plotcut is a cutcode element that tracks specific locations and can do calculations like whether this code is a raster or not.